### DISCUSSION TITLE
Lethal company AI.

### DISCUSSION THREAD

**Discussion by arandomuser0** [winstxnhdw/lc-hax#476] [Ideas]
Just like this one

**Comment by arandomuser0** [comment]
https://github.com/Synt-Axe/lethal-company-ai?tab=readme-ov-file

**Comment by winstxnhdw** [comment]
It’s cool but say goodbye to your FPS if this is added.

**Reply by arandomuser0** [reply]
not if you just make it to do simple tasks

**Reply by winstxnhdw** [reply]
Then there's not really a point is there? Maybe I don't see it, but what do you think would be useful?

**Reply by arandomuser0** [reply]
watch this https://www.youtube.com/watch?v=poZt_KjCwV4&t=263s

**Reply by winstxnhdw** [reply]
Yeah, honestly, I don't see the appeal of this yet.
